common/service/config/config.go

```diff
@@ -499,6 +501,18 @@ type (
 		// URI is the namespace default URI for visibility archiver
 		URI string `yaml:"URI"`
 	}
+
+	Security struct {
+		// Signing key provider for validating JWT tokens
+		JWTKeyProvider       JWTKeyProvider `yaml:"jwtKeyProvider"`
+		PermissionsClaimName string         `yaml:permissionsClaimName`
+	}
+
+	// Contains the config for signing key provider for validating JWT tokens
+	JWTKeyProvider struct {
+		KeySourceURIs []string      `yaml:"keySourceURIs"`
+		RefreshTime   time.Duration `yaml:"refreshTime"`
```

Comment:
can we use the term "interval" instead of "time" (that is the term we use for dynamic config refresh interval configuration)

Reply:
I like "interval". Renamed.